src/main/resources/templates/search.html

```diff
@@ -37,11 +37,11 @@ <h4>Search for your car</h4>
 		</select>
 </div>
 
- <div class="form-group" id="carYear">
+ <div class="form-group row" id="carYear">
 	        <label for="year">Year</label>
 	        <br>
 <!-- 	        <input class="form-control" type="number" th:field="*{year}" id="year" placeholder="Year"> -->
-	        <select name="year" id="year">
+	        <select name="year" id="year" class="form-control col-sm-4">

```

Comment:
Bootstrap has some nice looking UI components. Unless specific CSS is needed, I would recommend just using bootstrap classes for any given HTML component as it will clean up the UI immensely. This will also allow it to be flexible with a variety of screen sizes.